ethcore/src/spec/spec.rs

```diff
@@ -267,19 +273,46 @@ impl Spec {
 	}
 
 	/// Create a new Spec which conforms to the Frontier-era Morden chain except that it's a NullEngine consensus.
-	pub fn new_test() -> Self {
-		Spec::load(include_bytes!("../../res/null_morden.json") as &[u8]).expect("null_morden.json is invalid")
-	}
+	pub fn new_test() -> Spec { load_bundled!("null_morden") }
 
 	/// Create a new Spec which is a NullEngine consensus with a premine of address whose secret is sha3('').
-	pub fn new_null() -> Self {
-		Spec::load(include_bytes!("../../res/null.json") as &[u8]).expect("null.json is invalid")
-	}
+	pub fn new_null() -> Spec { load_bundled!("null") }
 
 	/// Create a new Spec with InstantSeal consensus which does internal sealing (not requiring work).
-	pub fn new_test_instant() -> Self {
-		Spec::load(include_bytes!("../../res/instant_seal.json") as &[u8]).expect("instant_seal.json is invalid")
-	}
+	pub fn new_instant() -> Spec { load_bundled!("instant_seal") }
+
+	/// Create a new Olympic chain spec.
+	pub fn new_ethereum_olympic() -> Spec { load_bundled!("ethereum/olympic") }
+
+	/// Create a new Frontier mainnet chain spec.
+	pub fn new_ethereum_frontier() -> Spec { load_bundled!("ethereum/frontier") }
```

Comment:
I thought the best place would be `Spec` constructor methods instead of free functions, but I don't think these can be namespaced on the call site, thus the addition. Should they be made functions?